src/main/webapp/app/app.module.ts

```diff
@@ -27,6 +28,7 @@ import { ErrorComponent } from './layouts/error/error.component';
     AppRoutingModule,
     // Set this to true to enable service worker (PWA)
     ServiceWorkerModule.register('ngsw-worker.js', { enabled: false }),
+    NgMultiSelectDropDownModule,
```

Comment:
Do we need to import `NgMultiSelectDropDownModule` here? Isn't it enough to import in `therapeutical-regime.module`?

Reply:
Need to check, will have a look at it tomorrow

Reply:
You were right, it was only needed to be imported on therapeutical-regime.module.ts and it was corrected

Reply:
@andrenns Since this module will be used in other components, did you try to add it here and remove it from therapeutical-regime.module.ts?

Reply:
@DanielFran no need to import it to the whole app if it is only needed on a single place. If there's a necessity to add it in more places, add it there. This is a performance improvement.